db/migrate/20230512064613_remove_faculty_id_from_targets.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveFacultyIdFromTargets < ActiveRecord::Migration[6.1]
+  def up
+    remove_index :targets, :faculty_id, if_exists: true
+    remove_column :targets, :faculty_id, :integer
+  end
+
+  def down
+    add_column :targets, :faculty_id, :integer
+    add_index :targets, :faculty_id
+  end
```

Comment:
```suggestion
  def change
    remove_column :targets, :faculty_id, :integer, index: true
  end
```

Reply:
No, there are changes in the file.

Reply:
🤔

Reply:
Not here, but in the other two file, I have removed the `belongs_to` and `has_many` relationship.